src/helpers/nsToMs.ts

```diff
@@ -0,0 +1 @@
+export const nsToMs = (ns: number) => Math.round(ns / 10000) / 100;
```

Comment:
Could we add a ts comment or expand this method name to explain what types we are being converted? I assume nanoseconds to milliseconds.